README.md

```diff
@@ -166,6 +166,38 @@ cordova.plugins.firebase.auth.getCurrentUser().then(function(userInfo) {
 })
 ```
 
+### updateProfile()
+Updates the current user's profile data.
+
+Passing a `null` value will delete the current attribute's value, but not
+passing a property won't change the current attribute's value.
+
+```js
+cordova.plugins.firebase.auth.updateProfile({
+   displayName: "Jane Q. User",
+   photoURL: "https://example.com/jane-q-user/profile.jpg"
+}).then(function() {
```

Comment:
I'd prefer 
```js
cordova.plugins.firebase.auth.updateProfile({
     displayName: "Jane Q. User",
     photoURL: "https://example.com/jane-q-user/profile.jpg"
}).then(function() {
    console.log("user profile updated");
});
```
Because `currUser` is undefined in your case. Also the error callback is redundant too - we know how to catch promise rejections.

Reply:
Changed example in readme. Please review new change

The purpose was to call out the difference between null property and not providing a property.